app/models/concerns/match_decisions/route_thirteen_cancel_reasons.rb

```diff
@@ -0,0 +1,42 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchDecisions
+  module RouteThirteenCancelReasons
+    extend ActiveSupport::Concern
+
+    def pre_hsa_decision? = [
+      'MatchDecisions::Thirteen::ThirteenClientMatch',
+      'MatchDecisions::Thirteen::ThirteenMatchAcknowledgement',
+    ].include?(type)
```

Comment:
WHAT?!?  This is not a method definition I've seen before.  Is this the same as?
```
def pre_hsa_decision? 
    [
       'MatchDecisions::Thirteen::ThirteenClientMatch',
       'MatchDecisions::Thirteen::ThirteenMatchAcknowledgement',
     ].include?(type)
end
```

Reply:
Definitely an improvement in the reasons setup below.  Thanks for implementing.

Reply:
good question, that must have snuck in from my C# background and just worked so I didn't even notice it was a different way of doing it.